Sources/Testcontainers/Docker/Models/ImageParams.swift

```diff
@@ -14,6 +14,18 @@ public struct ImageParams {
     public let src: String?
     public let repo: String?
     
+    public init(image: String, src: String?, repo: String?) throws {
+        let items = image.split(separator: ":")
+        guard let name = items.first, let tag = items.last else {
```

Comment:
Hmm I think this may have an issue- `items.first` and `items.last` can be the same item if there's just a single item in the array.

Reply:
Yess, you are rigth ! I'll take a spin around it later. 🤓🤓